docs_user/modules/openstack-tls_everywhere.adoc

```diff
@@ -0,0 +1,203 @@
+[id="tls-everywhere_{context}"]
+
+//:context: tls
+
+//kgilliga: This module will be converted to an assembly. Check xref contexts.
+
+= TLS Everywhere
+
+_Disclaimer: the below steps were reproduced on a FreeIPA 4.10.1 server. The location of files and directories may slightly change on different versions._
+
+== Prerequisites
+
+* Check that the source deployment is using TLS Everywhere, skip this step if it's not.
+* Make sure the previous Adoption steps have been performed successfully.
+* Make sure the backend services on the new deployment are not started yet.
+
+== Variables
+
+Define the shell variables used in the following steps. The values are illustrative and refer to a single node standalone director deployment. Use values that are correct for your environment:
+
+----
+IPA_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100 podman exec -ti freeipa-server-container"
+----
+
+In this example the FreeIPA instance is running on a separate host, in a container.
+
+== Transfer the CA certificate and key from FreeIPA to the cert-manager Operator
+
+=== Exporting from FreeIPA
+
+There are a couple of ways to do it:
+
+1. You can copy the certificate from the `/etc/pki/pki-tomcat/alias` directory. The file is named `ca.crt`.
+2. You can interact with the database located in the `/etc/pki/pki-tomcat/alias` directory ("*db" files).
+
+==== Locating the CA certificate and key
+
+List all the certificates inside your NSSDB:
+
+----
+$ $IPA_SSH certutil -L -d /etc/pki/pki-tomcat/alias
+----
+
+The `-L` option lists all certificates, and `-d` specifies where they are stored. This will produce some output like this:
+
+----
+Certificate Nickname                                         Trust Attributes
+                                                             SSL,S/MIME,JAR/XPI
+
+caSigningCert cert-pki-ca                                    CTu,Cu,Cu
+ocspSigningCert cert-pki-ca                                  u,u,u
+Server-Cert cert-pki-ca                                      u,u,u
+subsystemCert cert-pki-ca                                    u,u,u
+auditSigningCert cert-pki-ca                                 u,u,Pu
+----
+
+The item you need to consider is the first one: `caSigningCert cert-pki-ca`.
+
+==== Exporting the items (certificate and key)
+
+The command below generates a P12 file with both, the certificate and the key. The `/etc/pki/pki-tomcat/alias/pwdfile.txt` file contains the password that protects the key. You can use it to both, extract the key and generate the new file, `/tmp/freeipa.p12`. You can also choose another password. Should you choose to apply a different password for the new file, simply replace the parameter of the `-w` option, or alternatively use the `-W` (capital W) option followed by the password (in clear text).
```

Comment:
I would suggest that you instead just supply the example where you create a pk12 file with a different password.
Its generally not a good idea to reuse passwords like you have done here.

So - 1) create password file
        2) extract using pk12util

Reply:
Ignore this last comment.  Looks like you never actually copy the pk12 file out of the IPA server.  Thats great.